docs/index.md

```diff
@@ -27,13 +27,8 @@ You can follow one of the quickstart guides below to set up a Kubernetes cluster
 
 ## Post-install configuration
 
-After successfully installing Codacy, go follow the post-installation configuration steps:
+After successfully installing Codacy, follow the post-install configuration steps:
 
 *  [Use External Databases](configuration/external-dbs.md)
 *  [Logging](configuration/logging.md)
 *  [Monitoring](configuration/monitoring.md)
-*  Configure one or more of the following supported git providers:
```

Comment:
This configuration is now done at the end of the install.md, when the user first accesses the Codacy UI and is presented with the onboarding.